app/views/hosts/_operating_system.html.erb

```diff
@@ -23,7 +23,9 @@
 
 <div id='root_password'>
   <%= password_f f, :root_pass, :help_inline => _("Password must be 8 characters or more"), :required => true,
-                 :unset => action_name == "edit", :keep_value => action_name == "clone"  %>
+                 :unset => action_name == "edit", :keep_value => action_name == "clone",
+                 :placeholder => (@host.root_pass.empty?) ? nil : "**********",
+                 :help_inline => (@host.root_pass_source.present?) ? _("The password is set by %s param.") % @host.root_pass_source : nil %>
```

Comment:
`param` is a bit confusing since it's not actually a parameter. How about:
```suggestion
                 :help_inline => (@host.root_pass_source.present?) ? _("The password is currently set by %s.") % @host.root_pass_source : nil %>
```
Also, should we have some way of hiding or changing the help text once the user edits the field?